package.json

```diff
@@ -11,39 +11,39 @@
   "author": "Adobe Inc.",
   "private": true,
   "dependencies": {
-    "@adobe/aio-lib-ims": "^6.5.0",
```

Comment:
You can also add the engines.node schema to this package.json. This will require the nodejs version on any npm command...

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines

```json
{
  "engines": {
    "node": ">=20.0.0"
  }
}
```

Reply:
Unless you have any concerns, I'm going to set the minimum required version to ">=18". Currently, it is 
```
"node": "^14.18 || ^16.13 || >=18"
```
This will be in line with maintained NodeJS versions (See https://nodejs.org/en/about/previous-releases)